File: solutions/026.remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let oldNumsLength = nums.length
    if(oldNumsLength<=1){
        return oldNumsLength
    }
    let lastNum = nums[0]
    let result = 1
    for(let i=1;i<oldNumsLength;i++){
        if(lastNum !== nums[i]){
            nums[result]= nums[i]
            result++
            lastNum = nums[i]
        }
    }
    return result
    
    
};

File: solutions/026.remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int counter = 0;
        for(int i=0;i<nums.length;i++){
            if(i==0 || nums[i]!=nums[counter-1]){
                nums[counter]=nums[i];
                counter++;
            }
        }
        return counter;
    }
}